amp/amp-service-worker-offline-commenting.js

```diff
@@ -0,0 +1,49 @@
+/* global ERROR_MESSAGES, SITE_URL */
+{
+	const queue = new wp.serviceWorker.backgroundSync.Queue( 'amp-wpPendingComments' );
+	const errorMessages = ERROR_MESSAGES;
+
+	const commentHandler = ( { event } ) => {
+		const clonedRequest = event.request.clone();
+		return fetch( event.request )
+			.then( ( response ) => {
+				return response;
```

Comment:
Is there any need for this `then` handler now? Should it not just do:

```js
fetch( event.request ).catch( () => {
```

In reality a success `response` should always be a 302 redirect, correct? That is, unless AMP live list commenting is being used in which case it would be a 200 or 202 response. 

In particular, there is the [`handleResponse`](https://github.com/xwp/pwa-wp/blob/e63139548142aa261584902dcd22129457121658/wp-includes/js/service-worker-navigation-routing.js#L16-L57) function. 

Given that the 302 response is irrelevant for streaming or app shell, we don't need any of that logic. But some 500 error handling would perhaps be nice, but not essential for now. So I think we can just eliminate the `then`.